### ISSUE TITLE
Porting to Hugo

### ISSUE THREAD

**Issue by melroy89** [zquestz/bitcoincash#116]
As agreed let's migrate to an off-the-self static site generator Hugo.

For both now and later, this would help the development of the site, since developers know Hugo better. And is more standardized, allowing us to focus more on the content rather than the build.

**Comment by zquestz** [comment]
Agreed, closing due to inactivity.

**Comment by melroy89** [comment]
See here the code: https://gitlab.melroy.org/bitcoincash/bitcoincash.org

**Comment by zquestz** [comment]
Oooh very cool, didn't know this was already started. Let me reopen!

**Comment by melroy89** [comment]
Yeah! Well.. @zquestz you were AFK for about 1 full year!? So.. How are you now? What happened?

Anyway, glad you are back behind the keyboard 🤣

**Comment by zquestz** [comment]
BTW, just dumped Juicer. Stack is much easier to setup now.

**Comment by melroy89** [comment]
> BTW, just dumped Juicer. Stack is much easier to setup now.

Does that mean we can now run on the latest ruby version again?

**Comment by melroy89** [comment]
> > BTW, just dumped Juicer. Stack is much easier to setup now.
> 
> Does that mean we can now run on the latest ruby version again?

I will answer my own question, but somewhat: https://github.com/zquestz/bitcoincash/commit/71f5be225ba95da0128d17b144e81c0151a6377e

But not fully.. the latest Ruby version is v4 now.
